README.md

```diff
@@ -92,6 +104,14 @@ A complete NVRAM backup and restore can be performed between similar devices and
      Perform a backup before upgrading and restore it after to preserve your settings.
      You will experience some routing issues while the coordinator rebuilds its routing table.
  - CC2531 backups can only be restored to CC2531 devices running similar firmware versions.
+ - To run "zigpy_znp.tools.nvram_read" and "zigpy_znp.tools.nvram_write" you need to have Python installed on your system as well as required dependencies.
+   - If you are running a Linux or macOS based OS you need to have python and pip installed on your system. If you do not have them installed, refer to your distribution package manager to get it set up. (On Debian/Ubuntu; `sudo apt update && sudo apt-get install python3-pip` should work. `sudo pip` ... is not the optimal way of installing packages but Python package management is out of scope for this document.)
+     - Download and extract zigpy-znp using `wget -O cc2538-bsl.zip https://codeload.github.com/zigpy/zigpy-znp/zip/master && unzip zigpy-znp.zip`
+   - If you are running a Windows based OS download [Python for Windows](https://www.python.org/downloads/) and install. After installation verify Python is correctly installed by running `python -V` in the Command Prompt (cmd.exe). It should return Python and the version number. 
+     - Download the [zipped code](https://github.com/zigpy/zigpy-znp/archive/master.zip) and extract to a folder, then run the wanted "zigpy_znp.tools.nvram_read" and "zigpy_znp.tools.nvram_write" commands from the `\zigpy-znp-master\zigpy_znp\tools\` directory via the command-line/prompt.
+       - If you receive a message similar to `Python is not recognized as an internal or external command, operable program or batch file.`, it means that Python is either not installed or the system variable PATH has not been set. You will need to launch Python from the folder in which it is installed or adjust your system variables to allow it to be launched from any location.
+   - To install the required dependencies "pyserial" and "intelhex", open command-line/prompt and check if `pip` is installed by running `pip -V` which will show its version and install location. Then From the same command-line/prompt run the command `pip3 install pyserial intelhex`.
```

Comment:
Using `pip` is the only correct way to install zigpy-znp and its required dependencies. Manually installing them will pull in incompatible versions of packages (i.e. `pyserial-asyncio==0.5` on Windows).

Reply:
Hopefully fixed now